humble/ros_packages.txt

```diff
@@ -29,3 +29,4 @@ ros-humble-udp-driver
 ros-humble-udp-msgs
 ros-humble-velodyne-pointcloud
 ros-humble-yaml-cpp-vendor
+ros-humble-pacmod3-msgs
```

Comment:
we also need it on the noetic folder

Reply:
actually no this is not related to that, its for ros2

Reply:
This is a ros2 package, so not required in noetic?